//test exception 8

int main() {
  int a;
  int b;
  int c;
  a = 4;
  b = 8;
  c = b - (2 * a);
  return a / c;
}
